uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/Moment.java

```diff
@@ -73,5 +75,8 @@ default Date asDate(){
 
         @JsMethod(namespace = JsPackage.GLOBAL)
         public static native Moment moment();
+
+        @JsMethod(namespace = JsPackage.GLOBAL)
+        public static native Moment moment(String dateStr, String pattern);
```

Comment:
@nmirasch would it also work if you use these different constructors?
- https://momentjs.com/docs/#/parsing/unix-timestamp-milliseconds/
- https://momentjs.com/docs/#/parsing/date/
I guess these ones do not force you to format the date....